src/components/Summary/InterviewSummaryModal/index.js

```diff
@@ -199,166 +126,78 @@ const InterviewSummaryModal = props => (
                         interviewer={comment.author}
-                        questions={[questions]}
+                        questions={[questions[0]]}
                         comments={[comment]}
                       />
-                    );
-                  }
-                  comment_count = comment_count + 1;
+                    </>
+                  );
+
                   return questioncomment;
                 })}
-                <Typography.Title level={4}>
-                  Comments by questions
-                </Typography.Title>
+                <br></br>
+                {new_score}
+                <br></br>
+                <Typography.Title level={4}>Comments</Typography.Title>
                 <Row type="flex" justify="space-around">
                   {records.map(record => {
+                    let wrap_data = [];
+                    const columns = [
+                      {
+                        title: 'Author',
+                        dataIndex: 'Author',
+                        key: 'Author',
+                        sorter: (a, b) => a.Author.localeCompare(b.Author),
+                        sortDirections: ['descend', 'ascend'],
+                      },
+                      {
+                        title: 'Content',
+                        dataIndex: 'Content',
+                        key: 'Content',
+                      },
+                      {
+                        title: 'Time',
+                        dataIndex: 'Time',
+                        key: 'Time',
+                        sorter: (a, b) => a.Time.localeCompare(b.Time),
+                        sortDirections: ['descend', 'ascend'],
+                      },
+                    ];
+                    const onChange = (pagination, filters, sorter, extra) => {
+                      console.log('params', pagination, filters, sorter, extra);
+                    };
                     const single_question = record.ques;
-                    const single_history = record.history.items;
-                    const single_comments = single_history.map(
-                      x => x.snapComments,
-                    );
-                    let all_comments = '';
-                    const single_comments_2 = single_comments.map(x => x.items);
-                    const single_comments_3 = single_comments_2.map(x =>
-                      x.map(y =>
-                        all_comments.concat(
-                          y.author,
-                          '  :  ',
-                          y.content,
-                          '  ',
-                          '\n',
-                        ),
-                      ),
-                    );
+                    const wrap_table = record.history.items
+                      .map(x => x.snapComments)
+                      .map(y => y.items)
+                      .flat()
+                      .map(z => {
+                        wrap_data.push({
```

Comment:
why not just returning like this
```javascript
const wrap_data= record.history.items
                      .map(x => x.snapComments)
                      .map(y => y.items)
                      .flat()
                      .map(z => 
                        ({
                          Author: z.author,
                          Content: z.content,
                          Time: z.time.substring(11, 16),
                        })
                      )
                      .sort((a, b) => a.Time.localeCompare(b.Time));

```

There's no need to create one more variable.